### PR TITLE
Update restful_api.md fix OOM starting serve api

### PR DESCRIPTION
**Pull request by G4mot**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily receiving feedbacks. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

Please describe the motivation of this PR and the goal you want to achieve through this PR.

## Modification

Please briefly describe what modification is made in this PR.

## BC-breaking (Optional)

Does the modification introduce changes that break the backward-compatibility of the downstream repositories?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit tests to ensure the correctness.
3. If the modification has a dependency on downstream projects of a newer version, this PR should be tested with all supported versions of downstream projects.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

File: docs/en/serving/restful_api.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ The user can open the http url print by the following command in a browser.
 - **Please check the http url for the detailed api usage!!!**
 
 ```shell
-lmdeploy serve api_server ./workspace --server-name 0.0.0.0 --server-port ${server_port} --tp 1
+lmdeploy serve api_server ./workspace --server-name 0.0.0.0 --server-port ${server_port} --tp 1  --cache-max-entry-count 0.1
```
Comment (AllentDan) [inline docs/en/serving/restful_api.md:12]: may leave it as default. Just remind users that more arguments can be found through `-h`

**Comment by lvhan028** [comment]
Hi, @G4mot 
Could you please fix the linting error?
```shell
pip install pre-commit
cd lmdeploy
pre-commit install .
pre-commit run --all-files
```

**Review by AllentDan — COMMENTED** [review]
May update Chinese document as well. Besides, if possible, the description can be refined through LLM models such as ChatGPT.

**Comment by lvhan028** [comment]
Hi, @G4mot OOM issue has been fixed. FAQ has updated it too.

**Pull request closed without merging**